src/fides/api/models/privacy_notice.py

```diff
@@ -24,6 +22,20 @@
 )
 
 
+class Language(Enum):
+    """Using BCP 47 Language Tags
+
+    TODO propose adding this list:
+    https://www.techonthenet.com/js/language_tags.php
+    """
+
+    en_us = "en_us"  # US English
+    en_gb = "en_gb"  # British English
```

Comment:
Adrian brought up BCP 47 which looked good to me, I was thinking of adding underscored lowercase versions of this format here?